docs/reference/setup/install/docker.asciidoc

```diff
@@ -48,7 +48,7 @@ Elasticsearch can be quickly started for development or testing use with the fol
 
 ["source","sh",subs="attributes"]
 --------------------------------------------
-docker run -p 9200:9200 -e "http.host=0.0.0.0" -e "transport.host=127.0.0.1" {docker-image}
+docker run -p 9200:9200 {docker-image} -E "http.bind_host=0.0.0.0" -E "http.publish_host=127.0.0.1" -E "transport.host=127.0.0.1"
```

Comment:
I also checked with @clintongormley that `http.publish_host`, if undefined, follows the `transport.host` setting, not `http.host`, so the existing example should provide the same results.